### PR TITLE
Add docs for Ralph operators

### PR DESCRIPTION
**Pull request by Lbqds**

### CODE REVIEW

File: docs/ralph/getting-started.md [hunk 1]
```diff
@@ -170,6 +170,73 @@ Contract Foo() {
 }
 ```
 
+## Operators
+
+### Arith operators
```
Comment (h0ngcha0) [inline docs/ralph/getting-started.md]: [Nit] Perhaps use `Arithmetic operators`

**Review by h0ngcha0 — APPROVED** [review]
LGTM :+1:

File: docs/ralph/getting-started.md [hunk 2]
```diff
@@ -170,6 +170,73 @@ Contract Foo() {
...
+
+### Comparison operators
+
+There are inequality operators such as `<`, `>`, `<=`, and `>=` available for `I256` and `U256` types, and equality operators `==`, `!=` for all primitive types.
+
+Note that the comparison operators cannot be used with array types or struct types.
+
+### Bit operators
+
+Ralph supports bit operators such as `&`, `|`, `^`, `<<`, and `>>`, here are some examples:
```
Comment (polarker) [inline docs/ralph/getting-started.md]: bit operators only work for `U256` type

**Review by polarker — APPROVED** [review]
LGTM, just one minor remark

**Pull request merged**